src/components/ContinuationIn/SummaryBusinessHomeJurisdiction.vue

```diff
@@ -228,14 +212,15 @@
             <v-col
               cols="12"
               sm="9"
+              class="file-buttons pt-2 pt-sm-0"
```

Comment:
This and below: fixed vertical alignment, ie

![image](https://github.com/user-attachments/assets/f9f2f6cb-2019-4f6e-a437-ceb9b974c59f)